riotfile.py

```diff
@@ -586,6 +609,7 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
         # 3.0     3.6, 3.7, 3.8
         # 3.1     3.6, 3.7, 3.8
         # 4.0     3.8, 3.9, 3.10
+        # 4.1     3.8, 3.9, 3.10, 3.11
```

Comment:
The django docs linked below actually say that it's 4.2 that adds support for 3.11.